ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main.yaml

```diff
@@ -14,6 +14,31 @@ tests:
   commands: make lint
   container:
     from: src
+
+- as: e2e-aws-ran-profile
+  optional: true
+  steps:
+    cluster_profile: aws-telco
+    env:
+      BASE_DOMAIN: commatrix-ci.devcluster.openshift.com
```

Comment:
I suppose you don't have this domain registered? Please use the standard `telco5g-ci.devcluster.openshift.com`